tests/techmap/kogge-stone.tcl

```diff
@@ -0,0 +1,14 @@
+yosys -import
+
+read_verilog +/choices/kogge-stone.v
+read_verilog lcu_refined.v
+design -save init
+
+for {set i 1} {$i <= 16} {incr i} {
+    design -load init
+    chparam -set WIDTH $i
+    yosys proc
+    equiv_make -blacklist ppa.nomatch lcu _80_lcu_kogge_stone equiv
```

Comment:
The fact that this matches on internal names makes me think we are using the wrong command to check equivalence here

Reply:
looks like the folklore is `opt_clean` before `equiv_make`